libs/util/app-config/src/lib/app-config.ts

```diff
@@ -175,8 +175,9 @@ export function loadAppConfig() {
               BACKGROUND_COLOR: parsedThemeSection.background_color,
               THUMBNAIL_PLACEHOLDER:
                 parsedThemeSection.thumbnail_placeholder ||
-                'assets/img/fallback.webp',
-              HEADER_BACKGROUND: parsedThemeSection.header_background,
+                'assets/img/thumb_placeholder.webp',
+              HEADER_BACKGROUND:
+                parsedThemeSection.header_background || 'var(--color-gray-500)',
```

Comment:
Originally the default value was in the datahub app, but hardcoded as well. I'm not really sure where the default value should go, because different apps might need different values but for now we only have one app using it. Here I put a very neutral background so I figured I could just put it in the app-config.

Anyway this can be changed later on without impacting the app configuration schema, so I'm not worried :)